site/components/pages/Home/index.tsx

```diff
@@ -65,16 +68,19 @@ export const Home: NextPage<Props> = (props) => {
 
       <Section variant="gray" className={styles.heroSection}>
         <div className="hero_container">
-          <div className="hero_newsBanner">
-            <Text t="button" align="end">
-              <Trans>📰 Recent Post: </Trans>
-            </Text>
-            <Link href="/blog/a-new-face-for-klimadao">
-              <a>
-                <Trans>A New Face For KlimaDAO</Trans>
-              </a>
-            </Link>
-          </div>
+          {props.latestPost && (
+            <div className="hero_newsBanner">
+              <Text t="button" align="end">
+                <Trans>📰 Latest News: </Trans>
+              </Text>
+              <Link href={`/blog/${props.latestPost.slug}`}>
+                <a>
+                  <Trans>{props.latestPost.title}</Trans>
```

Comment:
you can remove the translate tags

Reply:
Is this because sanity will return content in the requested locale?

Reply:
Eventually, yes. Right now it doesn't. But Linguee uses babel macros at compile time-- since the CMs content is at runtime I don't think we can translate it with linguee